### DISCUSSION TITLE
Governor limit proposal for Aptos

### DISCUSSION THREAD

**Discussion by sleepycorgi1** [wormhole-foundation/wormhole#2508] [Governance] [answered]
This is a discussion about modifying the governor limit on Aptos. The current daily limit is 5M/500k. The daily notional limit is currently close to being reached. 

Welcome any feedback from guardians on what amount to increase will be appropriate. 

I am going to start with a suggestion between 15-20M.

**Comment by tbjump** [comment]
Thanks for raising this issue! Looking at the [Dashboard](https://wormhole-foundation.github.io/wormhole-dashboard/), the Governor "fast transaction limit" seems to be exhausted with one VAA (sequence number [10097](https://wormhole-v2-mainnet-api.certus.one/v1/signed_vaa/22/0000000000000000000000000000000000000000000000000000000000000001/10097)) already enqueued. 

According to our internal monitoring, the transfer activity seems to be legitimate, i.e. Portal seems to remain fully collateralized. The transfer activity consists mostly of USDC transfers. 

Raising the Governor limit would expose the Token Bridge to a higher security risk if there was a vulnerability on the Aptos blockchain. Even though a young chain, Aptos itself has had a good security track record so far and the Wormhole integration has received [two external audits](https://github.com/wormhole-foundation/wormhole/blob/main/SECURITY.md). 

The assets currently locked on Aptos [consist mostly of USDC](https://wormhole.com/stats/). 

From a security perspective, I would support raising the Governor fast-transaction limit to a maximum of $20M/2M notional to meet user demand and re-assess the situation if demand falls back down. 

I would appreciate feedback from Guardians if they would adopt a $20M/2M setting for Aptos if it were to become part of the reference implementation, or alternatively which limits they seem to be appropriate.

**Comment by tbjump** [comment, marked as answer]
It seems like the activity on Aptos has returned to normal levels and a Governor change is not warranted anymore.
